File: src/main/java/dev/anvilcraft/addon/template/data/lang/AddonLangHandler.java
package dev.anvilcraft.addon.template.data.lang;

import dev.anvilcraft.addon.template.AddonConfig;
import dev.anvilcraft.lib.v2.config.ConfigData;
import dev.anvilcraft.lib.v2.registrum.providers.RegistrumLangProvider;

public class AddonLangHandler {

    /**
     * 语言文件初始化
     *
     * @param provider 提供器
     */
    public static void init(RegistrumLangProvider provider) {
        ConfigData.readConfigClass(provider, AddonConfig.class);
    }
}

File: src/main/kotlin/dev/anvilcraft/addon/template/data/lang/AddonLangHandler.kt
package dev.anvilcraft.addon.template.data.lang

import dev.anvilcraft.addon.template.AddonConfig
import dev.anvilcraft.lib.v2.config.ConfigData
import dev.anvilcraft.lib.v2.registrum.providers.RegistrumLangProvider

class AddonLangHandler {
    companion object {
        /**
         * 语言文件初始化
         *
         * @param provider 提供器
         */
        fun init(provider: RegistrumLangProvider) {
            ConfigData.readConfigClass(provider, AddonConfig::class.java)
        }
    }
}